fsharp-backend/src/LibExecution/StdLib/LibInt.fs

```diff
@@ -164,21 +164,20 @@ let fns : List<BuiltInFn> =
       sqlSpec = NotYetImplementedTODO
       previewable = Pure
       deprecated = NotDeprecated }
-    // ; { name = fn "Int" "power" 0
-    //   ; infix_names = ["^"]
-    //   ; parameters = [Param.make "base" TInt; Param.make "exponent" TInt]
-    //   ; returnType = TInt
-    //   ; description = "Raise `base` to the power of `exponent`"
-    //   ; fn =
-    //
-    //         (function
-    //         | _, [DInt base; DInt exp] ->
-    //             DInt (Dint.pow base exp)
-    //         | args ->
-    //             incorrectArgs ())
-    //   ; sqlSpec = NotYetImplementedTODO
-    //     ; previewable = Pure
-    //   ; deprecated = NotDeprecated }
+    { name = fn "Int" "power" 0
+      parameters = [ Param.make "base" TInt ""; Param.make "exponent" TInt "" ]
```

Comment:
Can you add the infix for `^` as well and add tests for it?

Reply:
This pull request is almost completed.
I have other question; for the infix, I added the functions here
![image](https://user-images.githubusercontent.com/13671857/101272404-b6d85e80-3740-11eb-8586-c4b6edf77b6c.png)
How should I be testing this? Thank you 🙏